ui/app/routes/-unload-model-route.js

```diff
@@ -0,0 +1,25 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+/**
+ * For replacing unload-model-route mixin without converting to modern type.
+ * Takes array of model names which should be unloaded when moving away from
+ * the route. Automatically unloads all `capabilities` and `control-group`
+ * data before unloading the specified models.
+ */
+export default class UnloadModelRoute extends Route {
```

Comment:
Is this needed or could we just use the decorator on each route as needed?

Reply:
Since we still have some class-based routes that use other mixins, I didn't want to force us to refactor everything for this change. In practice I couldn't get the decorators to play nice with the older routes, so that's why there's both here. Would love to move to decorators only in the future but not sure that's feasible for right now

Reply:
Could the decorator be applied to this route then?